input/file.js

```diff
@@ -1,7 +1,6 @@
 const Tail = require('tail').Tail
-const nginx = require('../format/nginx.js')
 
-function create ({name = 'File', path, parse = nginx.parser()} = {}) {
+function create ({name = 'File', path, parse} = {}) {
```

Comment:
Why would you default to `{}` ?
IMHO default only makes sense if they can actually be of help to run the function omitting some parameters.
Here path and parse are mandatory parameters, defaulting to `{}` to me give the impression this function can/should work if receiving an empty object as the argument

Reply:
@jduthon Good catch, it should have been removed.